plugins/github-actions/src/components/WorkflowRunDetails/WorkflowRunDetails.tsx

```diff
@@ -175,14 +175,17 @@ export const WorkflowRunDetails = ({ entity }: { entity: Entity }) => {
   const jobs = useWorkflowRunJobs({ hostname, owner, repo });
 
   const error = projectName.error || (projectName.value && details.error);
```

Comment:
I'm a little confused by the existing code, shouldn't it be something like:

```ts
const error = projectName.error ?? details.error;
const loading = projectName.loading ?? details.loading;
```

Reply:
The error and loading checks could stay the same as they are. The issue is due to asynchronous code that has `projectName.loading` and `details.loading` set to false, while the `details.error` was set because it was passed undefined values when `projectName` was still loading.

Reply:
I've explained the bug in more detail below.

Reply:
I feel the most straight-forward fix for this issue is to make the `useProjectName` synchronous. I don't see any reason for being async as is. This does also seem to be the case for at least a couple of the other CI/CD plugins btw. If the fix is simple enough, is it something you could apply to the other plugins as well where needed?